.drone.star

```diff
@@ -879,7 +877,6 @@ def uiTestPipeline(ctx, filterTags, early_fail, runPart = 1, numberOfParts = 1,
                  ocisServer(storage, accounts_hash_difficulty) +
                  waitForSeleniumService() +
                  waitForMiddlewareService() +
-                 restoreBuildArtifactCache(ctx, "testing_app", dirs["testing_app"]) +
```

Comment:
Can we remove `TESTING_DATA_DIR` at line 889?

Reply:
Also, `cloneCoreRepos()` has `git clone` of `https://github.com/owncloud/testing.git`
Can that be removed?